includes/tracker.php

```diff
@@ -162,9 +165,11 @@ public static function send_tracking_data( $override = false ) {
 	 * @since 1.0.0
 	 * @access public
 	 * @static
+	 * @deprcated since 3.6.0
 	 */
 	public static function is_allow_track() {
-		return 'yes' === get_option( 'elementor_allow_tracking', 'no' );
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( 'Tracker::is_allow_track', '3.6.0', 'Site_Usage_Opt_In::is_on()' );
+		return Site_Usage_Opt_In::is_on();
```

Comment:
Is it 3.5.0 or 3.6.0?

```suggestion
		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( 'Tracker::is_allow_track()', '3.6.0', 'Site_Usage_Opt_In::is_on()' );
		return Site_Usage_Opt_In::is_on();
```